lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -2139,6 +2139,19 @@ using namespace Js;
         return TRUE;
     }
 
+    JavascriptArray* JavascriptOperators::IterableToList(RecyclableObject* items, ScriptContext* scriptContext) {
```

Comment:
As far as I can see this is only used once, could you just do the steps inline without making a new method for it?